### PR TITLE
Travel component

### PR DESCRIPTION
**Pull request by JulianJendryka**
plz merge

### CODE REVIEW

File: frontend/src/app/travel/travel.component.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  };
+
+  input = {
+    startPlace: new FormControl(),
+    endPlace: new FormControl(),
+    startDate: new FormControl(),
+    endDate: new FormControl(),
+  };
+
+  ngOnInit(): void {
```
Comment (maxrohleder) [inline frontend/src/app/travel/travel.component.ts:43]: i dont like this

**Review by maxrohleder — APPROVED** [review]
Looks confusing but works so f it

**Comment by maxrohleder** [comment]
totto bene

**Comment by JulianJendryka** [comment]
review

**Pull request merged**